### DISCUSSION TITLE
umi项目不支持在css文件中使用tailwind的`@apply`指令？

### DISCUSSION THREAD

**Discussion by gmqiyue** [umijs/umi#12524] [General]
我使用pnpm create umi aproject创建的新项目，通过`pnpm exec umi g tailwindcss`添加配置后，src目录下的所有css文件使用`@apply`指令，构建出的代码没有被解析转译（如下图：
![image](https://github.com/umijs/umi/assets/87467755/b153f08d-d7f5-4b2b-b4b8-003def48289b)

**Comment by gmqiyue** [comment]
dev环境也是如此
![image](https://github.com/umijs/umi/assets/87467755/2e012410-c6a7-488b-b016-5411a3cf8042)

但是在tsx文件中配置tailwindcss样式类名是可以生效的
![image](https://github.com/umijs/umi/assets/87467755/384cb2c2-be08-4953-a4b0-efdaa3b8c9d2)

**Comment by fz6m** [comment]
目前内置 tailwind 和 unocss 功能都是比较老的时期做的，经过时间太久了，用的人也很少，现在都处于不可用状态。

如需使用，请参照 [tailwind](https://tailwindcss.com/docs/installation/using-postcss) 官方文档改用 postcss 插件或者 webpack 插件等形式使用，如果后续 umi 有计划了再完善内置的。

**Reply by gmqiyue** [reply]
嗯嗯好的，我说怎么都不讨论这个问题...建议在文档中备注一下暂不维护呢

**Reply by xlboy** [reply]
`unocss` 这边能引入 `@unocss/webpack` 来在 umi webpack 中使用吗，但似乎有一定的问题…
umi 这边处理配置文件时，应该是使用 cjs 进行的，而最新版的 `unocss` 又全面拥抱 esm 了，此时 cjs, esm 就冲突了…不然就是将 `unocss` 降回 cjs 的版本来保持兼容了

**Comment by bbcvc** [comment]
解决了吗？

**Reply by mazhenjie233** [reply]
我在ice也有这个问题，使用postcss引入也无法编译

**Comment by WuChenDi** [comment]
It seems that the adaptation was not properly done;
it was observed that in the official instructions provided, the Ant Design components are being overridden by Tailwind's default styles.

https://umijs.org/en-US/docs/guides/generator#tailwind-css-configuration-generator

![image](https://github.com/user-attachments/assets/2cd3bb64-e5cb-499f-811e-a74c9f45279c)

**Reply by cd996** [reply]
I'm having same issue
